src/app/components/check-in/check-in.component.ts

```diff
@@ -18,7 +18,9 @@ export class CheckInComponent implements OnInit {
   }
 
   SubmitData(){
-    this.checkInObject.checkIn(this.pinValue).subscribe(response => {}, error=>{});;
+    this.checkInObject.checkIn(this.pinValue).subscribe(response => {
+      console.log(response)
+    }, error=>{});;
```

Comment:
Same comment as previous pull request, we need to implement a logger.

Reply:
x2